java/ql/test/stubs/apache-commons-lang3-3.7/org/apache/commons/lang3/StringUtils.java

```diff
@@ -0,0 +1,7 @@
+package org.apache.commons.lang3;
+
+public class StringUtils {
+    public static boolean isEmpty(final CharSequence cs) {
+        return cs == null || cs.length() == 0;
```

Comment:
Stubs shouldn't have function bodies (the point is they stand in for external, unanalysed code, so we should produce the same result if this unconditionally returns false vs. having its true definition as at present)

Reply:
Understood, always wondered the same.